bulla-contracts/tests/BullaFactoring.test.ts

```diff
@@ -181,7 +181,7 @@ test("it handles BullaFactoring v2 events", () => {
 
   log.info("✅ should create a InvoiceKickbackAmountSent event with correct claim ID", []);
 
-  const totalRefundAmount = BigInt.fromI32(9000);
+  const totalRefundAmount = BigInt.fromI32(9000000);
```

Comment:
hmm I would revert this. We would have never witnessed the issue if it wasn't for it.